dask/dataframe/core.py

```diff
@@ -919,6 +920,15 @@ def _bind_operator_method(cls, name, op):
         """ bind operator method like DataFrame.add to this class """
         raise NotImplementedError
 
+    def estimate_size(self, _p=16):
```

Comment:
If we go the `nunique(approximate=True)` route, it's pretty messy to provide any parameters at all.